src/shopify/graphql/getExampleHandles.ts

```diff
@@ -1,16 +1,17 @@
 import { cached } from "@/utils/cached"
 import getExampleHandlesQuery from "@/shopify/graphql/getExampleHandles.graphql?raw"
 
-export const [getExampleHandles, clearCache] = cached(async (root: string, amount = 12) => {
-  const endpoint = `${root}api/2025-10/graphql.json`
+const [getExampleHandlesCached, clearCacheInternal] = cached(async (shopBaseUrl: string) => {
```

Comment:
With current approach, both root and count are used as cache key. As a result, when adjusting the products slider control, multiple repeated API calls are triggered. With this approach, we fetch all products and store it in cache with shop URL as cache key.